workshop/content/30-python/20-create-project/100-cdk-init.md

```diff
@@ -30,48 +30,60 @@ Executing Creating virtualenv...
 
 # Welcome to your CDK Python project!
 
-You should explore the contents of this template. It demonstrates a CDK app with two instances of
-a stack (`HelloStack`) which also uses a user-defined construct (`HelloConstruct`).
+You should explore the contents of this project. It demonstrates a CDK app with an instance of a stack (`CdkWorkshopStack`)
+which contains an Amazon SQS queue that is subscribed to an Amazon SNS topic.
 
 The `cdk.json` file tells the CDK Toolkit how to execute your app.
 
 This project is set up like a standard Python project.  The initialization process also creates
-a virtualenv within this project, stored under the .env directory.  To create the virtualenv 
+a virtualenv within this project, stored under the .env directory.  To create the virtualenv
 it assumes that there is a `python3` executable in your path with access to the `venv` package.
 If for any reason the automatic creation of the virtualenv fails, you can create the virtualenv
 manually once the init process completes.
 
 To manually create a virtualenv on MacOS and Linux:
 
+'''
 $ python3 -m venv .env
+'''
 
 After the init process completes and the virtualenv is created, you can use the following
 step to activate your virtualenv.
 
+'''
 $ source .env/bin/activate
+'''
 
 If you are a Windows platform, you would activate the virtualenv like this:
 
+'''
 % .env\Scripts\activate.bat
+'''
 
 Once the virtualenv is activated, you can install the required dependencies.
 
+'''
 $ pip install -r requirements.txt
+'''
 
 At this point you can now synthesize the CloudFormation template for this code.
 
+'''
 $ cdk synth
+'''
 
 You can now begin exploring the source code, contained in the hello directory.
 There is also a very trivial test included that can be run like this:
 
+'''
 $ pytest
+'''
```

Comment:
Shouldn't these be backticks? This is a Markdown file?

Reply:
Yes. But those dont render properly with hugo (it starts and ends its codeblock formatting).

Reply:
Nvmd, found a md workaround.
Fixed